### ISSUE TITLE
Check if Include folders/files do exists (in case they are removed)

### ISSUE THREAD

**Issue by buhtz** [bit-team/backintime#1586]
I accidentally missed a  part of a backup for over one year. It is my fault. But I think it would have helped me if BIT would warn about such situations.

The folders `A` and `B` where in the Include list of my snapshot profile. And snapshots are done as scheduled.
I renamed the folder `B` to `C`.
But I forgot to modify the snapshot profile adapting this situation. Because of that `C` was not part of the snapshot.

I am not sure how to fix it. Maybe a) when opening a profile (in GUI and CLI) or b) when start taking a snapshot the include list should be checked of existence first and warn about missings.

**Comment by aryoda** [comment]
> how to fix it. Maybe
> a) when opening a profile (in GUI and CLI) or
> b) when start taking a snapshot the include list should be checked of existence first and warn about missings.

I think both should be done but the thing is the warning is not "pushed" to the user in case of scheduled backups.

Perhaps a permanent systray icon (#1370) with a dedicated icon for existing warnings and errors could be solution for desktop users. Server-only users need log shipping or emails anyhow but should know how to do that.

**Comment by buhtz** [comment]
Yeah, the question about how to know about errors in scheduled backups is always in my head. There is an old Issue (#1018) from myself about that and could be a part of the solution.

**Comment by buhtz** [comment]
We have a first solution in #1718 by Rafel. Have not tested or checked it in details.

- [X] A log message (to syslog) is created on every snapshot. I think this is the minimum.
- [X] Also it warns with a message box when a snapshot is triggered manually via the GUI.
- [ ] The systrayicon/notify warning is missing. Don't know how to trigger that for sure.

Other suggestion about a solution? Rafael needs a direction from us.

**EDIT**: I am undecided about it. Not sure how "urgent" such a situation is for most of our users.

**Comment by buhtz** [comment]
Just for documentation: The check is done in CLI and warnings are logged and (if available) shown in the systray. The GUI is not involved because of problems related to problems regarding BITs lack of robust interproces communication.
